src/test/scala/nl/grons/metrics/scala/ImplicitsSpec.scala

```diff
@@ -12,7 +12,8 @@ class ImplicitsSpec extends FunSpec {
   describe("Implicits") {
     it("brings the implicit conversion functionToMetricFilter into scope") {
       // sanity check:
-      """val metricFilter: MetricFilter = (_: String, _: Metric) => true""" shouldNot compile
+      // TODO This test fails in Scala 2.12.0-RC1
+      //"""val metricFilter: MetricFilter = (_: String, _: Metric) => true""" shouldNot compile
```

Comment:
This test fails in Scala 2.12 because SAM support has been introduced in Scala 2.12.
I don't know how to write testcase that covers Scala 2.12 and before in this case.